website/content/en/status/report-2023-04-2023-06/pf.adoc

```diff
@@ -1,4 +1,4 @@
-=== Pf Improvements
+=== pf Improvements
```

Comment:
This has already been discussed with the report submitters and they choose to capitalize Pf but not pfsync, so please leave it as it is.